test/evergreen.yml

```diff
@@ -1940,6 +1940,16 @@ tasks:
           # run for 24 hours ( 24 * 60 = 1440 minutes), don't stop at failed tests, use default config
           format_test_script_args: -t 1440
   
+  - name: format-stress-sanitizer-lsm-test
+    commands:
+      - func: "get project"
+      - func: "compile wiredtiger address sanitizer"
+      - func: "format test script"
+        vars:
+          test_env_vars: ASAN_OPTIONS="detect_leaks=1:abort_on_error=1:disable_coredump=0" ASAN_SYMBOLIZER_PATH=/opt/mongodbtoolchain/v3/bin/llvm-symbolizer
+          # Run for 1 hour (60 mins), and explicitly set data_source to LSM
```

Comment:
Any particular reason for running the test for 60 min. ?
Otherwise change LGTM

Reply:
It's a new task/test aiming to cover LSM in format stress test suite. Ideally the test is set to run for enough time to be able to catch failures, and not wasting too much testing cycle. 60-mins is simply a "random" value popped in my mind (considering most format stress runs in Jenkins takes 2-3 hrs), and I'm prepared to be convinced to adopt certain more sensible value :)